### ISSUE TITLE
Slider

### ISSUE THREAD

**Issue by joshwooding** [jpmorganchase/salt-ds#2885]
```[tasklist]
### Tasks
- [ ] #2450 
- [ ] #2452 
- [ ] #2451 
```

**Comment by mark-tate** [comment]
Zazu Goal: merge to RC, a known state, stakeholder review required

**Comment by mark-tate** [comment]
design review today

**Comment by tomhazledine** [comment]
* Pushed fixes following design feedback on Monday, waiting on sign-off from @bhoppers2008 
* Outstanding bug with event-listener not being removed on un-mount. *Paused for now to focus on #2886*
* New (?) bug for Salt site examples: examples overflow box on medium/small screen widths - *will raise separate ticket*

**Comment by tomhazledine** [comment]
* Design sign-off from Nav ✅ 
* Still need to address event-listener bug, but otherwise ready for dev review (it's a big PR)

**Comment by mark-tate** [comment]
Goal: push to core by EOS

**Comment by tomhazledine** [comment]
Addressing memory-leak bug with event-handling refactor.

**Comment by tomhazledine** [comment]
Refactoring event handling to cut out some nasty edge-cases and weird behavioural quirks. Expecting dev work to finish early next week

**Comment by mark-tate** [comment]
Goal: by end of 22/07/24 PR up and seeking reviewing  during Cappuccino, 
Chance for release to Core if we can get review

**Comment by navkaur76** [comment]
26/07 - design review carried out on storybook PR + site docs. A few interaction issues noticed with the range with input example. Review summarised in Design QA page in figma.

**Comment by tomhazledine** [comment]
Focus and tab issues flagged by @navkaur76 . Fixed and waiting for design review.

**Comment by tomhazledine** [comment]
Design issues addressed.
Working through dev feedback.

**Comment by mark-tate** [comment]
Cortado Goal:
PR Review and merged to Labs (wk1)
Identify some third-party feedback before merge to Core

**Comment by tomhazledine** [comment]
Changes made. Awaiting dev re-review

**Comment by origami-z** [comment]
Aug 12 - PR ready, 1 more review, aim for stakeholder review this week

**Comment by tomhazledine** [comment]
Still waiting on second review.

No luck finding stakeholders to review so far.

**Comment by origami-z** [comment]
Aug 16 - Could be good to merge to lab, leave there until a stakeholder can review

**Comment by mark-tate** [comment]
Closing this issue in favour of  https://github.com/jpmorganchase/salt-ds/issues/4018
